mitodl/micromasters
# Updated Programs page

#### What are the relevant tickets?
Fix #370 
#### What's this PR do?
Use Nicolae's template for Program page. 
Add FAQs and Courses panels to Programs page. 
#### Where should the reviewer start?
cms/models.py
#### How should this be manually tested?
As an instructor: 
1)login to wagtail as a superuser 
2)Create a ProgramPage, fill out all the required fields. FAQs and Courses should appear as multifield panels, add a few.
As a user: Go to homepage, click on the link for any program. Explore the page with all three tabs populated with your data: Description, FAQs, Courses.
